CSharp/Library/Microsoft.Bot.Connector.Shared/CardEx.cs

```diff
@@ -87,5 +88,276 @@ private static Attachment CreateAttachment<T>(T card, string contentType)
                 ContentType = contentType
             };
         }
+
+        /// <summary>
+        /// Gets SSML markup for this <see cref="Attachment"/>
+        /// </summary>
+        /// <param name="attachment"></param>
+        /// <returns></returns>
+        public static string GetSpeech(this Attachment attachment)
+        {
+            if (attachment.Content is ReceiptCard)
+            {
+                return GetSpeech(attachment.Content as ReceiptCard);
+            }
+            else if (attachment.Content is SigninCard)
+            {
+                return GetSpeech(attachment.Content as SigninCard);
+            }
+            else
+            {
+                return GetStandardCardSpeech((dynamic)(attachment.Content));
+            }
+        }
+        /// <summary>
+        /// Gets SSML markup for this <see cref="ReceiptCard"/>.
+        /// </summary>
+        /// <param name="receiptCard"></param>
+        /// <returns></returns>
+        public static string GetSpeech(this ReceiptCard receiptCard)
+        {
+
+            var title = receiptCard?.Title;
```

Comment:
>? [](start = 35, length = 1)

it's not clear to sprinkle the coalesce operator everywhere - either it's null or not